lib/auth/auth_with_roles.go

```diff
@@ -3357,6 +3358,37 @@ func (a *ServerWithRoles) GetAuthPreference(ctx context.Context) (types.AuthPref
 	return a.authServer.GetAuthPreference(ctx)
 }
 
+// GetInstaller retrieves an installer script resource
+func (a *ServerWithRoles) GetInstaller(ctx context.Context) (types.Installer, error) {
+	if err := a.action(apidefaults.Namespace, types.KindInstaller, types.VerbRead); err != nil {
+		return nil, trace.Wrap(err)
+	}
+	installer, err := a.authServer.GetInstaller(ctx)
+	if err != nil {
+		if trace.IsNotFound(err) {
+			return installers.DefaultInstaller, nil
+		}
```

Comment:
I'd probably push it into actual auth server's GetInstaller method, not sure auth_with_roles which is an authz layer is the appropriate place for this fallback.